### ISSUE TITLE
Spelare - Jämförelsevy

### ISSUE THREAD

**Issue by VilijamRigo** [kvarak/htstatus#15]
Jag gillar att man kan öppna flera åt gången:
![image](https://user-images.githubusercontent.com/66732847/85224457-d9610000-b3ca-11ea-9a66-728b671373c2.png)

Skulle det gå att göra så att om man kan ha ett max antal kolumner? Så att man högst kan öppna 3-4 i bredd, sedan kommer de på raden nedanför?

Ex:
![image](https://user-images.githubusercontent.com/66732847/85224561-8e93b800-b3cb-11ea-9325-30e873d78370.png)

**Comment by kvarak** [comment]
Tack! Jag gillar det också :)

Som det är nu så används typ inbyggda saker så att det är enkelt att lägga till ett "kort", alla ligger dolda och jag bara tar fram de som man klickar på. Att göra om det så som du föreslår så kommer det krävas en helt annan lösning.

Men, om det är en enkelhet att jämföra, så kanske man kan visa det på något annat sätt? Om man gräver lite djupare för att ta reda på hur vi vill ha det, vet du vad du skulle vilja jämföra eller är det bara en bra sak i allmänhet som kanske kommer bli en bra sak?

**Comment by VilijamRigo** [comment]
Jag såg framför mig att det kunde vara ett sätt att klicka en hel uppställning, så att man kanske skulle kunna få plats med 10 spelare på en datorskärm. 
Men det jag framförallt är sugen på ar att kunna klicka på alla mina ex. träningsobjekt och kunna se dem på en gång.

**Comment by kvarak** [comment]
Ja, träningsobjekten kommer få en speciell sida på `/training` sen, med allt möjligt tänkte jag. Har inte allt (inte så mycket) klart i huvudet, men tänkte ta sånt dit. Men bra idéer! Markerar denna som en `/training` feature.

Skulle det funka för det du tänker?

**Comment by VilijamRigo** [comment]
Nej, gör inte det. Det var inte bara det jag ville ha. Mer antalet

Den sön 21 juni 2020 15:11Albert Rigo <notifications@github.com> skrev:

**Comment by VilijamRigo** [comment]
Jag skulle vilja kunna klicka på alla mina backar och titta på dem samtidigt, det var så jag menade. Det skulle då även kunna användas för träning. Men inte i huvudsak

**Comment by VilijamRigo** [comment]
Ändrade du prioritet för att det skulle vara för mycket jobb för hur det ser ut nu?

**Comment by kvarak** [comment]
> Jag skulle vilja kunna klicka på alla mina backar och titta på dem samtidigt, det var så jag menade. Det skulle då även kunna användas för träning. Men inte i huvudsak

Ok, då är jag med.

> Ändrade du prioritet för att det skulle vara för mycket jobb för hur det ser ut nu?

Ja, för det kommer innebära massa jobb :/
Men när de "viktigaste" sakerna är på plats, kan man gå tillbaks till dessa.

**Comment by kvarak** [comment]
Released in v3.23
